.vscode/tasks.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Can the vscode launch configuration and setting be moved into a separate PR?

Reply:
I think so, yes. I'll try to split it up 👍

Reply:
yes, this should be possible.